release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1723,6 +1722,82 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
++
+As an example, the `wait-for` output when an invalid boot disk is used is now:
+[source,terminal]
+----
+"level=info msg=Cluster has hosts requiring user input
+level=debug msg=Host master-1 Expected the host to boot from disk, but it booted the installation image - please reboot and fix boot order to boot from disk QEMU_HARDDISK drive-scsi0-0-0-0 (sda, /dev/disk/by-path/pci-0000:03:00.0-scsi-0:0:0:0)
+level=debug msg=Host master-2 Expected the host to boot from disk, but it booted the installation image - please reboot and fix boot order to boot from disk QEMU_HARDDISK drive-scsi0-0-0-0 (sda, /dev/disk/by-path/pci-0000:03:00.0-scsi-0:0:0:0)
+level=info msg=cluster has stopped installing... working to recover installation"
+----
+(link:https://issues.redhat.com/browse/OCPBUGS-4998[*OCPBUGS-4998*])
+
+* Previously, the assisted-installer-controller on the installed cluster would run continuously even after the cluster had completed installation. Because assisted-service runs on the bootstrap node and not on the cloud, and because the assisted-service goes offline after the bootstrap node reboots to join the cluster, the assisted-installer-controller was unable to communicate with assisted-service to post updates and upload logs and loops. In this version, the bug fix modifies the assisted-installer-controller to check the cluster installation without using assisted-service, and to exit when the cluster installation is complete. (link:https://issues.redhat.com/browse/OCPBUGS-4240[*OCPBUGS-4240*])
+
+* Previously, installing a cluster to the AWS Commercial Cloud Services (C2S) `us-iso-east-1` region failed with an error message stating an `UnsupportedOperation`. With this fix, installing to this region now succeeds. (link:https://issues.redhat.com/browse/OCPBUGS-2324[*OCPBUGS-2324*])
+
+* Previously, installations on AWS could fail because the installation program did not create the cloud.conf file with the necessary service endpoints in it. This led to the machine config operator creating an empty cloud.conf file that lacked the service endpoints, leading to an error. With this update, the installation program always creates the cloud.conf file so that the installation succeeds. (link:https://issues.redhat.com/browse/OCPBUGS-20401[*OCPBUGS-20401*])
+
+* Previously, if you installed a cluster using the Agent-based installer and your pull secret had a null "auth" or "email" field, the installation would fail without providing a useful error. With this update, the `openshift-install agent wait-for install-complete` command validates your pull secret and notifies you if there are null fields. (link:https://issues.redhat.com/browse/OCPBUGS-14405[*OCPBUGS-14405*])
```

Comment:
```suggestion
* Previously, if you installed a cluster using the Agent-based installer and your pull secret had a null `auth` or `email` field, the installation would fail without providing a useful error. With this update, the `openshift-install agent wait-for install-complete` command validates your pull secret and notifies you if there are null fields. (link:https://issues.redhat.com/browse/OCPBUGS-14405[*OCPBUGS-14405*])
```